examples/deps_lib/BUILD.bazel

```diff
@@ -23,10 +23,6 @@ npm_package(
     name = "deps_lib",
     srcs = [
         ":deps_ts",
-        # TODO: This seems undesirable to have to specify the transitive closure of ts_project
```

Comment:
yay

Reply:
yup. that one was a very big rabbit hole

Reply:
Can the TODO in examples/BUILD.bazel also be removed?

Reply:
Oh right. Thanks for pointing it out. Forgot about that one and it is a different use case that is also now fixed. Updated.